### DISCUSSION TITLE
🔗 Dark mode for the entire website (Duplicate)

### DISCUSSION THREAD

**Discussion by fabigrafo** [elementor/elementor#20503] [Feature Request]
I know there are some plugins that make this more or less possible and of course using CSS. But doesn't it make much more sense if you could set it directly in Elementor? Dark mode is increasingly used on a relatively large number of websites. Why is Elementor resisting it?

**Comment by aaliyahfraser** [comment]
You can easily do that with add-ons. Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. Here we talk about “**[Best Elementor Addons and Plugins for WordPress 2023 – Compared](https://quomodosoft.com/best-elementor-addons-and-plugins/)**”
